### PR TITLE
fill managedBy field for all services envs

### PR DESCRIPTION
**Pull request by wpjunior**

### CODE REVIEW

**Review by gvicentin — APPROVED** [review]
LGTM

File: app/app.go [hunk 1]
```diff
@@ -1567,13 +1567,28 @@ func (app *App) Envs() map[string]bindTypes.EnvVar {
...
+		envVar.ManagedBy = fmt.Sprintf("%s/%s", e.ServiceName, e.InstanceName)
+		mergedEnvs[e.Name] = envVar
 	}
 	sort.Strings(toInterpolateKeys)
 	for _, envName := range toInterpolateKeys {
 		tsuruEnvs.Interpolate(mergedEnvs, toInterpolate, envName, toInterpolate[envName])
 	}
 	mergedEnvs[tsuruEnvs.TsuruServicesEnvVar] = tsuruEnvs.ServiceEnvsFromEnvVars(app.ServiceEnvs)
+
+	mergedEnvs["TSURU_APPNAME"] = bindTypes.EnvVar{
```
Comment (crgisch) [inline app/app.go:1580]: just gotta make sure those are properly write-protected

**Review by crgisch — APPROVED** [review]
👍

File: app/app.go (hunk 1 above)
Comment (crgisch) [reply app/app.go:1580]: just checked and they [are](https://github.com/tsuru/tsuru/blob/main/api/app.go#L1175)

**Pull request merged**